vsclib/setup.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# (C) 2022 MBition GmbH
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+import setuptools
+
+long_description="""Library to create, validate, and generate VSC files
+This library can be used by translators parsing other IDL files (FrancaIDL, etc)
+to build up an equivalent VSC structure, make sure that all datatype can
+resolve, and generate a python dictionary that can be written out as
+as VSC YAML file."""
+
+setuptools.setup(
+    name="vsclib",
+    version="0.0.1",
+    author="Magnus Feuer",
+    author_email="magnus.feuer@mercedes-benz.com",
+    description="VSC Generator Library",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/COVESA/vsclib",
+    packages=setuptools.find_packages(),
+    scripts=["vsc_test.py" ],
+    data_files=[],
+    include_package_data=True,
+    classifiers=[
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "License :: Other/Proprietary License",
```

Comment:
As we use MPL2 - is the license statement here correct? Compare with e.g. https://github.com/mozilla/libmozdata/blob/master/setup.py

Also, here we explictly state Python 3.7 and 3.8, but in buildcheck we use Python 3.10. Is it needed to specify Python version here. (In the long run, but not now,  we should preferably perform tests with all Python versions we claim to support. I.e. if we say here that Python 3.7 is sufficient then we shall have a build that verifies that 3.7 actually is sufficient)

Reply:
> As we use MPL2 - is the license statement here correct? Compare with e.g. https://github.com/mozilla/libmozdata/blob/master/setup.py

I agree, it looks wrong to me.

> Also, here we explictly state Python 3.7 and 3.8, but in buildcheck we use Python 3.10. Is it needed to specify Python version here. (In the long run, but not now, we should preferably perform tests with all Python versions we claim to support. I.e. if we say here that Python 3.7 is sufficient then we shall have a build that verifies that 3.7 actually is sufficient)

Yeah the buildcheck has evolved over time.  Good catch, we should make this consistent and also avoid overspecifying such details.